lib/class-anvato-library.php

```diff
@@ -202,7 +202,7 @@ private function build_request_params( $args = array() ) {
 	 *     function because the same timestamp is needed more than once.
 	 * @return string The URL after formatting with sprintf().
 	 */
-	private function build_request_url( $params = array(), $time ) {
+	private function build_request_url( $params, $time ) {
```

Comment:
I know that is an array is not passed then it will throw a fatal error but did you checked that the is no further point of failure if a non array is passed? probably is best to do some parsing in the function just in case